src/shared-constants.h

```diff
@@ -66,6 +66,7 @@ extern Name SHARED;
 extern Name EVENT;
 extern Name ATTR;
 extern Name ASSIGN_GOT_ENTRIES;
+extern Name STACK_LIMIT;
```

Comment:
I don't think we need this in shared constants if it's for a single pass. Or do we expect to use it in more places?